all-in-one/dockerhub-description.yml

```
@@ -0,0 +1,22 @@
+# This uses the peter-evans workflow: https://github.com/peter-evans/dockerhub-description
+name: Update Docker Hub Short-Description & Overview
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - README.md
+      - .github/workflows/dockerhub-description.yml
+jobs:
+  dockerHubDescription:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Update Docker Hub Short-Description & Overview
+      uses: peter-evans/dockerhub-description@v3
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_PASSWORD }}
+        repository: peterevans/dockerhub-description
+        short-description: A back-end service and web-based UI for the OSCAL Editor.
```

This needs to point to your new BUILDING-TESTING.md
```suggestion
        username: ${{ secrets.DOCKERHUB_USERNAME }}
        password: ${{ secrets.DOCKERHUB_PASSWORD }}
        repository: peterevans/dockerhub-description
        short-description: A back-end service and web-based UI for the OSCAL Editor.
        readme-filepath: all-in-one/BUILDING-TESTING.md
```

---

Are we sure on that?  When someone lands at Docker Hub aren't they more likely to want to just use the image rather than re-build it?

---

Yeah I misunderstood the docs. Sorry @Bronstrom this is good as-is.